package.json

```diff
@@ -8,8 +8,10 @@
     "node": "4.1.x"
   },
   "scripts": {
-    "start": "node index.js",
-    "test": "mocha test/*_test.js test/**/*_test.js"
+    "start": "node -r babel-core/register -r dotenv/config lib/index.js",
+    "prepublish": "babel lib -d dist",
```

Comment:
I'd rather to use it in the entry file personally. I'm not a big fan of the `npm start` command here and I almost feel like its a little nicer to just be explicit about them in the entry file (index.js). See what others think?